Dockerfile

```diff
@@ -198,3 +198,34 @@ COPY --from=build-src /src/bottlerocket/agents/src/bin/migration-test-agent/ssm-
 COPY --from=build-src /usr/share/licenses/testsys /licenses/testsys
 
 ENTRYPOINT ["./migration-test-agent"]
+
+# =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^= =^..^=
+# Builds the Kubernetes Workload test agent image
+FROM public.ecr.aws/amazonlinux/amazonlinux:2 AS k8s-workload-agent
+ARG ARCH
+
+# TODO remove unzip once aws-cli moves out
+RUN yum install -y unzip iproute && yum clean all
+ARG AWS_CLI_URL=https://awscli.amazonaws.com/awscli-exe-linux-${ARCH}.zip
+
+# Copy aws-iam-authenticator
+COPY --from=tools /aws-iam-authenticator /usr/bin/aws-iam-authenticator
+COPY --from=tools /licenses/aws-iam-authenticator /licenses/aws-iam-authenticator
+
+# TODO move this out, get hashes, and attribute licenses
+# Download aws-cli
+RUN temp_dir="$(mktemp -d --suffix aws-cli)" && \
```

Comment:
Do you actually need the aws cli? This is the only place I see it.

Reply:
I'm not actually sure if this is used internally some how. Now that I've gotten further than when I added this, I'm guessing no. I copied this portion from the sonobuoy definition since these workload tests are basically the same thing, just executed differently. If it is not needed here, then it would not be needed their either and we should clean it up.

Reply:
@etungsten it looks like [you added this for the sonobuoy image](https://github.com/bottlerocket-os/bottlerocket-test-system/commit/f0f4604e2a201fdb12ea13f3088404c94cdfd29d). Do you know, or remember, if this is actually needed for something when running sonobuoy?

Reply:
EKS clusters kubeconfigs uses either `aws-cli` or `aws-iam-autenticator` to authenticate to the cluster and it expect those tools to be in path. See https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html#create-kubeconfig-manually.

Reply:
Awesome, thanks for the details Erikson. So since the workload agent testing is based on sonobuoy plugin usage I believe it needs the same things the sonobuoy agent does.

Reply:
That being said, now that there are two container images using aws-cli, we should move this to the `tools` image and copy the binary in here like we do for `aws-iam-authenticator` above.

Reply:
Add https://github.com/bottlerocket-os/bottlerocket-test-system/issues/675 to track moving aws cli to `tools` to keep that work separate from this PR.